sentry_sdk/profiler.py

```diff
@@ -0,0 +1,186 @@
+"""
+This file contains code from https://github.com/nylas/nylas-perftools, which is published under the following license:
```

Comment:
I would say "This file is originally based on code from ..." I think that's more accurate, yes? Otherwise could sound like this file is verbatim from nylas.